tests/integration/Models/UserFlagTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace MailPoet\Test\Models;
+
+use MailPoet\Models\UserFlag;
+
+class UserFlagTest extends \MailPoetTest {
+  
+  function testItCanCreateOrUpdate() {
```

Comment:
This test should be split into 3 different tests. So when it fails we will see what is wrong.